File: LinguagemC/EXERCICIO_SOMA_VETOR.c
#include <stdio.h>

int main()
{
    //Realiza soma e média do vetor
    
    int sizeNumber;
    
    printf ("Quantos numeros serao digitados: ");
    scanf ("%d", &sizeNumber);
    
    double vector[sizeNumber], sumNumbers, average;
    
    
    for (int i = 0 ; i < sizeNumber ; i++)
    {
       printf ("Digite um numero: ");
       scanf ("%lf", &vector[i]);
    }
    
    printf("Valores: ");
    for (int i = 0 ; i < sizeNumber ; i++)
    {
       printf ("%.1lf  ", vector[i]);
    }
    printf("\n");
    
    
    for (int i = 0 ; i < sizeNumber ; i++)
    {
       sumNumbers += vector[i];
    }
    
    
    average = sumNumbers / sizeNumber;
     
    printf("Soma: %.2lf\n", sumNumbers);
    printf("Media: %.2lf\n", average);
     
    return 0;
}

File: LinguagemPython/exercicioSomaVetor.py
# Realiza a soma do vetor e calcula a média de seus números

vectorSize: int
averageVectorNumbers: float
sumVectorNumbers: float

vectorSize = int(input("Quantos números serão digitados? "))

vectorNumbers: float = [0 for x in range(vectorSize)]

for i in range(0, vectorSize):
    vectorNumbers[i] = float(input(f"Número {i + 1}/{vectorSize}: "))

sumVectorNumbers = 0
for i in range(0, vectorSize):
    sumVectorNumbers += vectorNumbers[i]

averageVectorNumbers = sumVectorNumbers / vectorSize

print("Valores: ", end="")
for i in range(0, vectorSize):
    print(f"{vectorNumbers[i]:.1f} ", end="")

print()
print(f"Soma: {sumVectorNumbers:.2f}")
print(f"Média: {averageVectorNumbers:.2f}")
